### PR TITLE
feat(pp): handle `null` type case

### PR DESCRIPTION
**Pull request by yongjiajie**
[Notion link](https://www.notion.so/Passport-Scanner-Edge-Cases-null-8057773fada0480f90661edfa53fa6ac) 

This PR adds... 

- handling of `null` type case when scanning passport

- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
- [x] I've requested reviews from 1-2 reviewers
- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
- [x] I've added/updated unit/integration tests
- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
- [ ] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
- [ ] I've added documentation in README/[Notion](https://www.notion.so/82b92fb1007640328dab9582c0a3694e?v=3b6ce48202cf40ad8450553799b13146)

### CODE REVIEW

File: src/utils/passportScanning.test.ts [hunk 1]
```diff
@@ -88,14 +88,24 @@ describe("passportScanning", () => {
       });
 
       describe("cases without passportId", () => {
-        it("should return empty string if it does not contain passportId", () => {
+        it("should return empty string if data is null", () => {
```
Comment (andrehadianto) [inline src/utils/passportScanning.test.ts:91]: to match the `case with passportId`, should we also add test for data is just a string that's not `passportId` object?

```
event = {
  data: JSON.stringify("invalid-string")
};
```

Comment (andrehadianto) [reply src/utils/passportScanning.test.ts:91]: should have the same result with `passportId === undefined`

**Review by andrehadianto — CHANGES_REQUESTED** [review]
some addition?

File: src/utils/passportScanning.test.ts (hunk 1 above)
Comment (yongjiajie) [reply src/utils/passportScanning.test.ts:104]: Have added this test case!

**Review by andrehadianto — APPROVED** [review]
LGTM 🚀

**Pull request merged**